judgels-backends/sandalphon/sandalphon-api/src/main/java/judgels/sandalphon/api/submission/programming/SubmissionInfo.java

```diff
@@ -0,0 +1,14 @@
+package judgels.sandalphon.api.submission.programming;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import judgels.jophiel.api.profile.Profile;
+import org.immutables.value.Value;
+
+@Value.Immutable
+@JsonDeserialize(as = ImmutableSubmissionInfo.class)
+public interface SubmissionInfo {
+    long getSubmissionId();
```

Comment:
Nit: can we call this just `getId()`. Because this is already about `Submission`. To be consistent with `ProblemInfo`, `UserInfo`, etc.